### PR TITLE
Sprint 49

### PR DESCRIPTION
**Pull request by MiguelLZPF**
## Description

AWS KMS included as a custodial provider, minor fixes and updates

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue) 🐞
- [x] New feature (non-breaking change which adds functionality) ✨
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 💥
- [ ] This change requires a documentation update 📖

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. 🧪

- [x] Local Tests (npm run test)
- [ ] Manually (example: cli with wizard or web using web interface)
- [ ] Local Github Actions (act pull_request)

**Test Configuration**:

- Node version (latest LTS):
  - [ ] 16
  - [x] 18
  - [x] 20

### CODE REVIEW

**Review by M-Francia — DISMISSED** [review]

File: .github/workflows/all.testWithRpc.yml [hunk 1]
```diff
@@ -6,7 +6,9 @@ on:
 jobs:
   test-node:
     name: testing
-    runs-on: token-studio-linux-medium
+#    runs-on: [self-hosted, Linux, medium, ephemeral]
+    runs-on: ubuntu-latest
```
Comment (gregscullard) [inline .github/workflows/all.testWithRpc.yml:10]: Should be token-studio-linux-medium, it was updated recently by another PR

**Review by gregscullard — CHANGES_REQUESTED** [review]
Note change requested to github action, otherwise looks good

File: .github/workflows/all.testWithRpc.yml (hunk 1 above)
Comment (MiguelLZPF) [reply .github/workflows/all.testWithRpc.yml]: Ok! done

**Review by gregscullard — APPROVED** [review]
LGTM

**Review by rbarkerSL — DISMISSED** [review]

File: .github/workflows/all.testWithRpc.yml (hunk 1 above)
Comment (M-Francia) [reply .github/workflows/all.testWithRpc.yml:9]: with token-studio-linux-medium test dont pass. With ubuntu-latest all test pass, could we keep ubuntu-latest?

**Review by M-Francia — APPROVED** [review]

**Review by dalvizu — APPROVED** [review]

**Review by gregscullard — APPROVED** [review]
LGTM

**Pull request merged**